Kitodo-DataManagement/src/main/resources/db/migration/V2_54__update_LDAP_Group_table.sql

```diff
@@ -0,0 +1,32 @@
+--
+-- (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+--
+-- This file is part of the Kitodo project.
+--
+-- It is licensed under GNU General Public License version 3 or later.
+--
+-- For the full copyright and license information, please read the
+-- GPL3-License.txt file that was distributed with this source code.
+--
+
+-- 1. Update the dummy LDAP group
+
+UPDATE ldapGroup SET homeDirectory = '/home/test/users/{login}',
```

Comment:
Is this home directory path correct? In old (/home/{login}) and new (/usr/local/kitdodo/users/{login}) installations was never such a path existing. Why is this path added?